### ISSUE TITLE
Analyze the minimum value where powerlaw starts

### ISSUE THREAD

**Issue by thebooort** [geneura-papers/2019-ALife-Repos-PowerLaw#4]
Analyze the minimum value where powerlaw starts

**Comment by JJ** [comment]
Minimum what? We're not going to have the temporal evolution in all cases, although we can generate temporal stuff if needed.

**Comment by JJ** [comment]
Any feedback here?

**Comment by thebooort** [comment]
Yes, I am referring to the minimum value where powerlaw distribution becomes statistically significant to explain the data. If there is some "statiscially frequent" value we could be more accurate at choosing repositories that are relevant for the analysis. Even the maximum value ( if there is some heavy tail at the end) could be interesting

**Comment by JJ** [comment]
Ah, OK. Great. We hadn't done that before.

**Comment by thebooort** [comment]
![imagen](https://user-images.githubusercontent.com/18428931/52853739-c6765580-311c-11e9-8233-40aa992c02e7.png)

I have not seen any correlation at first. I am interested about any correlation between this value and the number of commits.

**Comment by JJ** [comment]
It can be correlated to anything. Including the language.
